internal/storage/storage_test.go

```diff
@@ -15,6 +15,7 @@ import (
 
 	rukpakv1alpha1 "github.com/operator-framework/rukpak/api/v1alpha1"
 	"github.com/operator-framework/rukpak/internal/util"
+	pkgstorage "github.com/operator-framework/rukpak/pkg/storage"
```

Comment:
Should we move this test into the public storage package?

Reply:
I tried but it causes import cycle.  I'll leave it here.
```
# github.com/operator-framework/rukpak/pkg/storage
package github.com/operator-framework/rukpak/pkg/storage
        imports github.com/operator-framework/rukpak/internal/storage: import cycle not allowed in test
FAIL    github.com/operator-framework/rukpak/pkg/storage [setup failed]
FAIL

```